tests/docker-compose.mariadb.yml

```diff
@@ -18,5 +18,6 @@ services:
       - "127.0.0.1:80:80"
   db:
     image: mariadb
+    command: mysqld --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci --character-set-client-handshake=FALSE
```

Comment:
Quick question: does the binary from mariadb is still named `mysqld`?

Reply:
yes, please see https://github.com/docker-library/mariadb/blob/master/10.3/Dockerfile#L143